regression/library/test_manycore_vcache_sequence.c

```diff
@@ -112,6 +112,56 @@ int test_manycore_vcache_sequence() {
                         
                         mismatch = mismatch || !data_match;
                 }
+
```

Comment:
I think these should be in a new regression test. With these, this test is verifying functionality of two things.

Reply:
They're functionally correct, and I like them. It doesn't make sense from an error-reporting point of view.

Reply:
I also think that we should better than just testing the base address.

Reply:
Can you file an issue?

Reply:
I think that Sasha's new test does more than this. I think it tests the high and low address of each vcache

Reply:
Different test with different consequences and different meaning.

Reply:
So, this one just does sequences. If you want sequences starting from *random* locations, the world is your oyster

Reply:
This thread is out of date.

Reply:
Join was called